nebari/template/stages/02-infrastructure/azure/modules/kubernetes/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "location" {
 }
 
 variable "resource_group_name" {
-  description = "name of qhub resource group"
+  description = "name of nebari resource group"
```

Comment:
There are mixed cases in various `*.tf` files - need to make sure all the references are `nebari` and not a mix of upper and lower case

Reply:
Again, I'm not seeing this... 
<img width="1460" alt="Screen Shot 2022-10-26 at 14 18 53" src="https://user-images.githubusercontent.com/42120229/198013060-a7ea1921-0806-4f78-b1b8-3cd71dee6741.png">